open-api/rest-catalog-open-api.yaml

```diff
@@ -191,7 +220,8 @@ paths:
 
     get:
       tags:
-        - Catalog API
+        - tables
+        - views
```

Comment:
hmm, this seems like a problem to me. I want to avoid overlapping tags if possible, because when this happens, you need to know "if I support tables, not views, do I support this API?". The answer here might be a simple yes, but for complex capabilities, it might become a case by case thing and things become really complex. What do you think?

Reply:
to be more clear, I am okay with for example LoadTable having both a tag for `tables` and `credentials-vending` since the second tag is dedicated to a specific feature in the API. But here, `tables` and `views` requires exactly the same thing, which is that this API exists. That is the problem.

Reply:
Having `tables`/ `views` here simply means that if a catalog supports `tables`**or** `views` then it needs to support this endpoint as you won't be able to do something with tables if namespaces aren't supported. See also https://github.com/apache/iceberg/pull/9940#issuecomment-2177235732.

Also I probably wouldn't tag any endpoint with `credentials-vending`, since that capability doesn't require to implement certain endpoints.